### PR TITLE
updated docs to reflect Envoy as a DS option

### PR DESCRIPTION
**Pull request by nvibert**
```release-note
updated docs to reflect Envoy as a DS option
```

### CODE REVIEW

File: Documentation/security/policy/language.rst [hunk 1]
```diff
@@ -827,10 +827,8 @@ latter rule will have no effect.
 .. note:: Layer 7 rules are not currently supported in `HostPolicies`, i.e.,
           policies that use :ref:`NodeSelector`.
 
-.. note:: Layer 7 policies will proxy traffic through an Envoy instance provided 
-          in each Cilium agent pod. As a result, L7 traffic targeted by policies 
-          depend on the availability of the Cilium agent pod. This includes L7 
-          policies (:ref:`proxy_visibility`).
+.. note:: Layer 7 policies will proxy traffic through a node-local :ref:`envoy` instance
```
Comment (learnitall) [inline Documentation/security/policy/language.rst]: Two suggestions here:

1. Do you mind rephrasing this sentence to remove the parenthesis? Parenthesis are great for extra, non-essential pieces of information or side comments, and I don't think they fit here.
2. Can we keep the comment that L7 traffic targeted by policies depends on the availability of the Cilium agent pod in the case that Envoy is embedded in the agent?

**Review by learnitall — CHANGES_REQUESTED** [review]
Good catch! Thanks! Couple questions for you.

File: Documentation/security/policy/language.rst (hunk 1 above)
Comment (nvibert) [reply Documentation/security/policy/language.rst]: Thanks for the feedback @learnitall - should be fixed now

File: Documentation/security/policy/language.rst [hunk 2]
```diff
@@ -827,10 +827,10 @@ latter rule will have no effect.
 .. note:: Layer 7 rules are not currently supported in `HostPolicies`, i.e.,
           policies that use :ref:`NodeSelector`.
 
-.. note:: Layer 7 policies will proxy traffic through an Envoy instance provided 
-          in each Cilium agent pod. As a result, L7 traffic targeted by policies 
-          depend on the availability of the Cilium agent pod. This includes L7 
-          policies (:ref:`proxy_visibility`).
+.. note:: Layer 7 policies will proxy traffic through a node-local :ref:`envoy`
+          instance, which will either be deployed as a DaemonSet or embedded in the agent pod.
+          When Envoy is embedded in the agent pod, Layer 7 traffic target by policies
```
Comment (learnitall) [inline Documentation/security/policy/language.rst]:
```suggestion
          When Envoy is embedded in the agent pod, Layer 7 traffic targeted by policies
```

**Review by learnitall — CHANGES_REQUESTED** [review]
One small grammar nit and then LGTM. Thanks!

**Review by learnitall — APPROVED** [review]

**Review by nathanjsweet — APPROVED** [review]

**Comment by mhofstetter** [comment]
MLH seems to be sleeping :( manually marking as `ready-to-merge`.

**Pull request merged**